crates/re_types/src/blueprint/components/name_ext.rs

```diff
@@ -0,0 +1,15 @@
+use super::Name;
+
+impl From<&str> for Name {
+    #[inline]
+    fn from(value: &str) -> Self {
+        Self(value.into())
+    }
+}
+
+impl From<String> for Name {
+    #[inline]
+    fn from(value: String) -> Self {
+        Self(value.into())
+    }
+}
```

Comment:
Codegen should output this for transparent strings (we do the same for components containing datatypes):

```
impl<T: Into<::re_types_core::ArrowString>> From<T> for Name {
    fn from(v: T) -> Self {
        Self(v.into())
    }
}
```

Reply:
https://github.com/rerun-io/rerun/issues/4536